governance/votes/Executive vote - December 10, 2021.md

```diff
@@ -0,0 +1,97 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### wstETH-A Parameter Changes
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmYuK441?network=mainnet#poll-detail), the following wstETH-A parameters will be updated if this executive proposal passes.
+
+* Increase the wstETH-A [Maximum Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/module-dciam) from 50 million DAI to 200 million DAI. 
+* Increase the wstETH-A [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam) from 3 million DAI to 20 million DAI.
+* Decrease the wstETH-A [Debt Ceiling Increase Cooldown (`ttl`)](https://makerdao.world/en/learn/governance/module-dciam) from 8 hours to 6 hours.
+* Decrease the wstETH-A [Auction Price Multiplier (`buf`)](https://makerdao.world/en/learn/governance/param-auction-price-multiplier) from 1.3 to 1.2.
+* Increase the wstETH-A [Local Liquidation Limit (`ilk.hole`)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 7 million DAI to 15 million DAI.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/wsteth-a-parameters-adjustment-dec-1st-2021/11934) to help inform your position before voting.
+
+### MATIC-A Parameter Changes
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmdzwZyS?network=mainnet#poll-detail), the following MATIC-A parameters will be updated if this executive proposal passes.
+
+* Increase the MATIC-A [Maximum Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/module-dciam) from 20 million DAI to 35 million DAI. 
+* Decrease the MATIC-A [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam) from 20 million DAI to 10 million DAI.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/matic-debt-increase-proposal-for-last-executive-of-2021/11989) to help inform your position before voting.
+
+### Replacing MKR Vesting Streams
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmYdDTsn?network=mainnet#poll-detail), the following MKR Vesting Streams will be canceled and replaced by streams funded by the Protocol Treasury (Pause Proxy) if this executive proposal passes.
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** starting July 1, 2021 through July 1, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
```

Comment:
A trifle, but I'd use "Oracles Core Unit (ORA-001)" instead, which 1) I think is what we've been using the most, and 2) kind of makes a bit more sense because ORA-001 is the codename for the Oracles Core Unit.

Reply:
don't know what I was doing there haha, fixed for all three.